packages/amplify-e2e-core/src/utils/projectMeta.ts

```diff
@@ -1,6 +1,16 @@
 import * as path from 'path';
 import * as fs from 'fs-extra';
 
+function getFeatureFlagConfig(projRoot: string) {
+  const featureFlagPath = path.join(projRoot, 'amplify', 'cli.json');
+  return JSON.parse(fs.readFileSync(featureFlagPath, 'utf8'));
+}
+
+function updateFeatureFlagConfig(projRoot: string, config: any) {
```

Comment:
It would be cleaner to collapse both of these functions into `setFeatureFlag(projRoot: string, feature: string, key: string, value: any)` and it handles reading, updating and writing the cli.json file internally

Reply:
I think we should have a set and get for this. In event we need to read from the config to determine what to change it to as opposed to overriding the config.